pre_commit/clientlib.py

```diff
@@ -323,6 +323,7 @@ def warn_unknown_keys_repo(
     cfgv.Optional('files', check_string_regex, ''),
     cfgv.Optional('exclude', check_string_regex, '^$'),
     cfgv.Optional('fail_fast', cfgv.check_bool, False),
+    cfgv.Optional('metrics_command', cfgv.check_string, None),
```

Comment:
I guess there's not really a place to update docs for new config stuff here, huh? Would be nice to explicitly say somewhere "we're gonna call this with the json as the last argument", but oh well 🤷

Reply:
Yeah, looks like all the docs are on the website and not in-repo. x_x